types/leaflet-side-by-side/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for leaflet-side-by-side 2.2
+// Project: https://github.com/digidem/leaflet-side-by-side
+// Definitions by: Gabriel O. Martins <https://github.com/OliverBiel/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Minimum TypeScript Version: 5.0
```

Comment:
```suggestion
```
IMO this is not needed (there is no language features from TS5.0 that would require this constraint)